kotlinlib/src/mill/kotlinlib/KotlinModule.scala

```diff
@@ -1,11 +1,12 @@
 /*
- * Copyright 2020-Present Original lefou/mill-kotlin repository contributors.
+ * Original code copied from https://github.com/lefou/mill-kotlin
+ * Original code published under the Apache License Version 2
+ * Original Copyright 2020-20024 Tobias Roeser
```

Comment:
```suggestion
 * Original Copyright 2020-2024 Tobias Roeser
```

same for this copyright header in other files